docker-compose.yml

```
@@ -226,4 +405,5 @@ volumes:
   localstack-data:
   redis-data:
   grafana-data:
-  prometheus-data: 
\ No newline at end of file
+  prometheus-data:
+  celerybeat-data: 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)